core/src/auctions.ts

```diff
@@ -127,7 +135,42 @@ export const enrichAuctionWithPriceDrop = async function (auction: Auction): Pro
     }
 };
 
-export const enrichAuctionWithPriceDropAndMarketValue = async function (
+export const enrichAuctionWithMarketData = async function (auction: Auction, network: string): Promise<Auction> {
+    if (!auction.isActive || auction.isFinished || !auction.collateralToCoverDebt || !auction.approximateUnitPrice) {
+        return auction;
+    }
+    try {
+        const marketData = await getMarketData(network, auction.collateralSymbol, auction.collateralToCoverDebt);
+        const suggestedMarketId = (await getBestMarketData(marketData)).marketId;
```

Comment:
You might want to use all other keys from here to populate other values, instead of calculating it one more time inside `enrichAuctionWithMarketValues` – which not only unnecessarily, but can return different values, which will be very confusing